docker/linux-386/Dockerfile

```diff
@@ -22,3 +23,7 @@ RUN apt-get update && \
     rm -rf /tmp/*
 
 ENV PATH="/root/sdk/${GO_VERSION}/bin:${PATH}"
+
+# Static FIPS build only supports clang.
```

Comment:
Is that accurate? Or some special requirement for aws-lc-rs?

For example: https://github.com/aws/aws-lc/blob/main/tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml#L58-L66
Same for Arm dimensions.

Reply:
We can remove this comment now :). We hadn't fixed the FIPS build for all compilers at the time.